recipes/warrant-lite/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "warrant-lite" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 16e9d6a2c9599f7a345877addbe2ed8203b76db6ded942cc5d6f77ab4ec6c49e
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip ==18.1
```

Comment:
This is very specific version for pip, can this be loosened at all?

Reply:
Without pinning pip I was getting this error: `pip._vendor.pep517.wrappers.BackendUnavailable`. 
I tried pinning pip based on this issue https://github.com/pypa/pip/issues/6164.

Reply:
Sure, but can it be `<19`?

Reply:
yes it can 👍